docs/source/pumped_langevin_sde.rst

```diff
@@ -0,0 +1,49 @@
+The Pumped Langevin Dynamics Solver
+=====================================
+
+Stochastic differential equations
+----------------------------------
+
+Pumped Langevin dynamics, which is a modification of a typical Langevin dynamics (Khosravi et al 2022), can be described by the stochastic differential equations (SDE)
+
+.. math::
+
+    d c_{i} = \Big[\big(-1+p-c_i^2\big)c_i-\lambda\partial_{i} f(\mathbf{c}) \Big]dt + \sigma dW_{i}\;\;\text{with}\;\;c_{i}(0)=0\;\;\forall i=1,\ldots,N
+
+where, :math:`p(t) = p_0 t/T` and :math:`\lambda` and :math:`\sigma` are hyperparameters controlling the strength of the gradient and the diffusion terms, respectively. Apart from typical Langevin equation, the preceding equation is augmented with additional term :math:`
+\big(-1+p-c_i^2\big)c_i` to represent three physical processes (Khosravi et al 2022). Similar to the Langevin solver, we have implemented a solver for the pumped-Langevin solver.
+
+Note that the total evaluation time is arbitrary and can be defined as :math:`T = n_\mathrm{iter}\times dt`. 
```

Comment:
This sentence seems a bit unrelated to the content above. Perhaps we can flesh it out a bit more and lead into the topic. I've attempted below.
Also, what do we mean by the evaluation time being arbitrary? I removed that reference since I believe that the time equation we have chosen has some meaning.

Please read this for correctness, as I am not confident that I have captured the intended meaning.

```suggestion
The total evaluation time of the physical pumped Langevin solver may differ from the time taken to run the simulation of it. Hence, the physical evaluation time can be defined as :math:`T = n_\mathrm{iter}\times dt`, where :math:`dt` is specific to the iteration time of a given machine.
```

Reply:
Wait, after reading the next paragraph, I am certain my edit here was wrong.

What does the evaluation time represent and why are we mentioning it?

Reply:
Yes you are right. We should keep it as it is. Because we have a time dependence in the equation with `p(t)` including the final time `T`.

Reply:
Oh, I see. I overlooked the T there but that makes more sense now!

In that case, my suggestion is a bit different: let's move this line up a bit. I will comment a suggestion on the line I think we should move it to :)

Reply:
If I'll do the change you suggested above, then the meaning still changes. I am insisting we should keep it as it is.

Reply:
Oh no, forget about the change attached to this thread, I agree it is wrong. I included another comment on a different line

Reply:
It keeps the meaning of your version